tests/hooks_test.py

```diff
@@ -209,3 +209,28 @@ def test_text_unicode_replacement_char_positive(s):
 )
 def test_text_unicode_replacement_char_negative(s):
     assert not HOOKS['text-unicode-replacement-char'].search(s)
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        '    .. warning:',
+        '.. warning:',
+        '.. warning: :',
+        '.. warning:::',
+        '.. warning::::',
```

Comment:
are these mistakes people actually make? I wouldn't bother with these and just stick to the original one-colon-is-an-error

Reply:
OK, sure, have gone back to the original regular expression you suggested